packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/dedicated-server.routing.js

```diff
@@ -12,8 +12,18 @@ export default /* @ngInject */ ($stateProvider) => {
             'billing:autorenew2016Deployment',
             'dedicated-server:cluster',
           ]),
-        isMultiAZAvailable: /* @ngInject */ (featureAvailability) =>
-          featureAvailability?.isFeatureAvailable('dedicated-server:cluster'),
+        hasMultiAZServers: /* @ngInject */ (iceberg) =>
+          iceberg('/dedicated/cluster')
+            .query()
+            .expand('CachedObjectList-Pages')
+            .execute(null, true)
+            .$promise.then(({ data }) => data.length !== 0),
```

Comment:
**nitpick:** Avoid making an API call just to decide if user has clusters. If it cannot be avoided, atleast re-use this response to display the Clusters listing page.